lib/src/values/logic_value.dart

```diff
@@ -733,6 +733,13 @@ abstract class LogicValue {
   /// invalid, [neq] will return `x`.
   LogicValue neq(dynamic other) => _doCompare(other, (a, b) => a != b);
 
+  /// Power operation.
+  ///
+  /// This will return a [LogicValue] of some input 'base' to the power of other
+  /// input 'exponent'. If one of the two input values is invalid, [pow] will
+  /// return ‘x’ of input size width.
+  LogicValue pow(dynamic other) => _doMath(other, _binPower);
```

Comment:
Should we change the name to `exponent` instead of `other`?  You can also reference in the comment as `[exponent]`